packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -63,7 +63,7 @@ export default function BookmarkFeedLayout({
   const shareBookmarksButton = (style: string, text?: string) => (
     <Button
       className={style}
-      icon={<SourceIcon />}
+      icon={<SourceIcon className="icon" />}
```

Comment:
Can you check what are the styles applied for using the icon class? I think this doesn't do anything anymore but I could be wrong.

Reply:
I do not remember exactly, but it applies some font-size (doesn't really matter it this case) and some margin, so icon is not so close to the text